using-timescaledb/telemetry.md

```diff
@@ -99,25 +110,58 @@ Alternatively, in a `psql` console, run:
 ALTER [SYSTEM | DATABASE | USER] { *db_name* | *role_specification* } SET timescaledb.telemetry_level=off
 ```
 
-If `ALTER DATABASE` is run, then this will disable telemetry for the specified
-database, but not for other databases in the instance. If `ALTER SYSTEM` is
-run, this will disable telemetry for the entire instance.
-Note that superuser privileges are necessary to run `ALTER SYSTEM`.
+If `ALTER DATABASE` is run, then this will turn down the telemetry
+report level for the specified database, but not for other databases
+in the instance. If `ALTER SYSTEM` is run, this will turn down the
+telemetry report level for the entire instance.  Note that superuser
+privileges are necessary to run `ALTER SYSTEM`.
+
+To remove the telemetry job for a database, connect to the database
+using `psql` and execute:
+
+```sql
+DELETE FROM _timescaledb_config.bgw_job
+WHERE job_type = 'telemetry_and_version_check_if_enabled'
+```
+
+>:TIP: To ensure that newly created databases do not use telemetry,
+>remove the telemetry job from the `template1` database.
```

Comment:
This should be removed, as this only applies to a subset of installations.

Reply:
Well, the docker image install timescaledb into the template database as well, so I think this is good to mention. I do agree that it does not apply generally though, so we should probably qualify it.

Reply:
Removed the mention instead.